tools/deployment/package-helm/templates/_helpers.tpl

```diff
@@ -0,0 +1,165 @@
+{{/*
+Expands the name of the chart.
+
+@return {string} The chart name (truncated to 63 characters)
+*/}}
+{{- define "clp.name" -}}
+{{- default .Chart.Name .Values.nameOverride | trunc 63 | trimSuffix "-" }}
+{{- end }}
+
+{{/*
+Creates a default fully qualified app name. We truncate at 63 chars because
+some Kubernetes name fields are limited to this (by the DNS naming spec). If
+release name contains chart name it will be used as a full name.
```

Comment:
```suggestion
Creates a default fully qualified app name. We truncate at 63 chars because some Kubernetes name
fields are limited to this (by the DNS naming spec). If release name contains chart name it will be
used as a full name.
```